hadoop-hdds/erasurecode/src/main/java/org/apache/ozone/erasurecode/rawcoder/ByteBufferEncodingState.java

```diff
@@ -97,6 +101,9 @@ void checkBuffers(ByteBuffer[] buffers) {
       }
 
       if (buffer.remaining() != encodeLength) {
+        if (LOG.isDebugEnabled()) {
+          LOG.debug("buffer remaining is " + buffer.remaining() + " encodelength is " + encodeLength);
```

Comment:
Please use the placeholder syntax for adding values into log messages, eg:

```
LOG.debug("buffer remaining is {} encodelength is {}", buffer.remaining(), encodeLength);
```
Also, when the parameters to the log message are simple ones, like above, then you do not need to wrap the `LOG.debug` call in `if (LOG.isDebugEnabled())`, as that happens already inside the Logger code. Therefore please also remove the IF statement surrounding the `LOG.debug` call.

Reply:
ok

Reply:
@sodonnel please review, thanks